deployments/epp/staging/deployment-scaledobject.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: epp-import-worker-epp
+spec:
+  scaleTargetRef:
+    name: epp-import-worker-epp
+    namespace: epp--staging
```

Comment:
This isn't a valid property:

```
> kubectl explain ScaledObject.spec.scaleTargetRef
GROUP:      keda.sh
KIND:       ScaledObject
VERSION:    v1alpha1

FIELD: scaleTargetRef <Object>


DESCRIPTION:
    ScaleTarget holds the reference to the scale target Object

FIELDS:
  apiVersion    <string>
    <no description>

  envSourceContainerName        <string>
    <no description>

  kind  <string>
    <no description>

  name  <string> -required-
    <no description>
```

The ScaledObject can only control objects in the same namespace as itself. So moving this to the ScaledObject metadata is what you wanted.